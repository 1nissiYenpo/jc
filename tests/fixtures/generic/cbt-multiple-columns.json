[{"key":"foo","cells":{"bat":{"bar":"baz"},"foo":{"bar1":"baz1","bar2":"baz2"}}}]
