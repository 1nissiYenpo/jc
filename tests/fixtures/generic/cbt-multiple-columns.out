----------------------------------------
foo
  foo:bar1                                 @ 1970/01/01-01:00:00.000000
    "baz1"
  foo:bar2                                 @ 1970/01/01-01:00:00.000000
    "baz2"
  bat:bar                                  @ 1970/01/01-01:00:00.000000
    "baz"